vmdb/app/controllers/api_controller.rb

```diff
@@ -95,6 +96,10 @@ def initialize
     @api_token_mgr   = TokenManager.new(@module)
   end
 
+  def self.all_encrypted_attributes
+    (ATTR_TYPES[:encrypted] | ::Vmdb::ConfigurationEncoder::PASSWORD_FIELDS).sort
```

Comment:
They are accessible via provisioning request options hash and configurations via virtual attribute of servers. We just need to make sure we cover all cases.  Thanks.